app/src/main/res/layout/lists_with_add_content.xml

```diff
@@ -59,6 +59,20 @@
                 android:layout_width="match_parent"
                 android:layout_height="44dp"
                 />
+
+            <com.joanzapata.iconify.widget.IconTextView
+                android:id="@+id/voice_entry_icon"
```

Comment:
Let's move all these XML attributes into a reusable style.

Reply:
That sounds good but why not we make a separate layout and include wherever we want using the include tag?